mac/deploy_mac.sh

```diff
@@ -93,6 +92,28 @@ build_installer_image()
 
 }
 
+brew_install_pinned() {
+    local pkg="$1"
+    local version="$2"
+    local pkg_version="${pkg}@${version}"
+    local brew_bottle_dir="${HOME}/Library/Cache/jamulus-homebrew-bottles"
+    local formula="/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Formula/${pkg_version}.rb"
+    echo "Installing ${pkg_version}"
+    mkdir -p "${brew_bottle_dir}"
+    pushd "${brew_bottle_dir}"
+    if ! find . | grep -qF "${pkg_version}--"; then
+        echo "Building fresh ${pkg_version} package"
+        brew developer on  # avoids a warning
+        brew extract --version="${version}" "${pkg}" homebrew/cask
+        brew install --build-bottle --formula "${formula}"
+        brew bottle "${formula}"
+        # uninstall again to ensure that we can install the package freshly without leftovers afterwards:
+        brew uninstall "${pkg_version}"
```

Comment:
Something like "since brew install doesn't run postinst scripts, we are uninstalling it before later on installing it again [...]"